### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: doc/POST_INSTALL.md [hunk 1]
```diff
@@ -0,0 +1 @@
+You now need to create the first admin user for __APP__. (FIXME: how...?)
```
Comment (alexAubin) [inline doc/POST_INSTALL.md:1]: The old disclaimer did not really explain how people are supposed to "create the first admin user" x_x Doesn't prevent merging I guess, but would be nice to clarify ...

**Review by alexAubin — APPROVED** [review]

**Comment by tituspijean** [comment]
!testme

File: scripts/upgrade [hunk 2]
```diff
@@ -75,24 +33,12 @@ then
 	ynh_script_progression --message="Upgrading source files..." --weight=1
 
 	# Download, check integrity, uncompress and patch the source from app.src
-	ynh_setup_source --dest_dir="$final_path" --keep=".env"
+	ynh_setup_source --dest_dir="$install_dir" --keep=".env nc" --full_replace
```
Comment (tituspijean) [inline scripts/upgrade:36]: Somehow this does not clean out the old NodeJS stuff (`Dockerfile .gitignore index.js noco-daily.js node_modules .npm package.json package-lock.json README.md .env.example`), so the webapp gets confused and still prompts for an upgrade even though it is up-to-date.

**Review by tituspijean — COMMENTED** [review]
Emails (one for the super admin, and invite) are not sent out.

**Comment by Salamandar** [comment]
!testme

File: scripts/upgrade (hunk 2 above)
Comment (Salamandar) [reply scripts/upgrade]: Yes, =1 was missing from --full_replace. Fixed.

**Pull request merged**